specification/indices/_types/IndexSettings.ts

```diff
@@ -70,6 +70,7 @@ export class RetentionLease {
 /**
  * @doc_id index-modules-settings
  * @ext_doc_id index-settings
+ * @ext_doc_id analyzer-update-existing
```

Comment:
> @ext_doc_id analyzer-update-existing

Unfortunately, you can have only one [externalDocs](https://spec.openapis.org/oas/latest#external-documentation-object) (and it can contain only one link).